src/engine/SCons/Platform/__init__.py

```diff
@@ -100,21 +101,15 @@ def platform_module(name = platform_default()):
             eval(full_name)
         else:
             try:
-                file, path, desc = imp.find_module(name,
-                                        sys.modules['SCons.Platform'].__path__)
-                try:
-                    mod = imp.load_module(full_name, file, path, desc)
-                finally:
-                    if file:
-                        file.close()
+                mod = importlib.import_module(full_name)
```

Comment:
do we lose the ability to specify the path?

Reply:
Well, the original construct was:
```python
file, path, desc = imp.find_module(name,                                        sys.modules['SCons.Platform'].__path__)
```
Which had the path hardcoded from ``SCons.Platform``. ``import_module(full_name)`` is essentially the same, isn't it? (it's ``full_name``, not ``name``).

Reply:
OK, let it be.